### PR TITLE
add jcf w/ npm auto-update

### PR DESCRIPTION
**Pull request by kennynaoh**
PR for #8725 
@pvnr0082t please help me review this pull request, thank you.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/w3co/jcf
 * Official website (optional, not the repository): https://www.psd2html.com/js-custom-forms/
 * NPM package url (optional): https://www.npmjs.com/package/jcf
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 107
   - Count of watchers: 13
   - Count of forks: 31
 * NPM download stats (optional):
   - Downloads in the last day: 48
   - Downloads in the last week: 143
   - Downloads in the last month: 446

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/jcf/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  "keywords": [
+    "jQuery",
+    "JCF",
+    "form",
+    "element",
+    "style"
+  ],
+  "author": {
+    "name": "psd2html"
+  },
```
Comment (pvnr0082t) [inline ajax/libs/jcf/package.json]: Let's use `"author": "psd2html"` here.

**Comment by pvnr0082t** [comment]
@kennynaoh The files in `v1.2.3` has already been minified. You don't need to minify again.

**Comment by pvnr0082t** [comment]
`v1.1.3` and `v1.0.3` contains other versions. Please check again.

**Comment by kennynaoh** [comment]
@pvnr0082t 
I have fixed problems above you mentioned.
Please help me review it again.
Thanks!

**Comment by pvnr0082t** [comment]
@kennynaoh OK, please rebase it on the latest master branch, thank you.

**Comment by kennynaoh** [comment]
@pvnr0082t 
I have rebased to the latest master branch.
Please help me review it again.
Thanks!

**Comment by pvnr0082t** [comment]
@PeterDaveHello I think this PR is ok. Please review it again, thank you.

**Comment by PeterDaveHello** [comment]
@kennynaoh as we discussed before, you should also mention the reason why to manually add the versions in the comment, not only in the commit log.

**Pull request merged**